ui/components/app/permission-cell/permission-cell.js

```diff
@@ -28,17 +28,28 @@ import { useI18nContext } from '../../../hooks/useI18nContext';
 import Tooltip from '../../ui/tooltip';
 import { PermissionCellOptions } from './permission-cell-options';
 
+function extractDomainOrOriginal(input) {
```

Comment:
Allows getting the hostname from the invoker (e.g. `https://app.uniswap.org` returns `app.uniswap.org`); maybe this is not the correct place for it

Reply:
You should not add any new changes to this file. To get the host we should use 
`const currentTabOrigin = useSelector(getOriginOfCurrentTab);`